java/org/contikios/cooja/Cooja.java

```diff
@@ -3303,7 +3303,7 @@ public void run() {
    * @throws UnsatisfiedLinkError
    *           If associated libraries could not be loaded
    */
-  public Simulation loadSimulationConfig(File file, boolean quick, Long manualRandomSeed)
+  public Simulation loadSimulationConfig(File file, boolean quick, long manualRandomSeed)
```

Comment:
This can be called with `manualRandomSeed` set as NULL via quickStartSimulationConfig(). If `manualRandomSeed` is NULL, no specified random seed has been specified and a new random seed should be generated for the simulation.